flocker/node/functional/test_docker.py

```diff
@@ -367,6 +367,23 @@ def added(_):
         d.addCallback(added)
         return d
 
+    def test_null_environment(self):
+        """
+        A container that does not include any environment variables contains
+        an empty ``environment`` in the return ``Unit``.
+        """
+        client = self.make_client()
+        name = random_name()
+        self.addCleanup(client.remove, name)
+        d = client.add(name, u"busybox:latest")
```

Comment:
Have added a custom image we can build at run time in the functional test, in the way we do with a couple of other tests. This is a small busybox-ish image but guaranteed to have no environment variables set.